bz-mill/config/definitions/map_namespace_to_query

```diff
@@ -0,0 +1,2 @@
+http://red.ht/2oiunCH bz.gmeno.ceph.new.customer.24hrs
+http://red.ht/2paAywM bz.gmeno.ceph.any.needinfo.24hrs
```

Comment:
it's saying that the BZ query is collecting ticket that have not changed in 24hrs